db/migrate/20190322140910_create_carts.rb

```diff
@@ -0,0 +1,9 @@
+class CreateCarts < ActiveRecord::Migration[5.2]
+  def change
+    create_table :carts do |t|
+      t.integer :user_id
```

Comment:
To add referential integrity here, you would change this line as follows:

```suggestion
      t.references :user, foreign_key: true
```

See https://guides.rubyonrails.org/active_record_migrations.html#foreign-keys 
and 
https://guides.rubyonrails.org/association_basics.html#creating-foreign-keys-for-belongs-to-associations.